airbyte-integrations/connectors/source-databend/source_databend/spec.json

```diff
@@ -0,0 +1,63 @@
+{
+  "documentationUrl" : "https://docs.airbyte.com/integrations/sources/databend",
+  "connectionSpecification" : {
+    "$schema" : "http://json-schema.org/draft-07/schema#",
+    "title" : "Source Databend Spec",
+    "type" : "object",
+    "required" : [
+      "host",
+      "username",
+      "database"
+    ],
+    "additionalProperties" : true,
+    "properties" : {
+      "host" : {
+        "title" : "Host",
+        "description" : "Hostname of the database.",
+        "type" : "string",
+        "order" : 0
+      },
+      "port" : {
+        "title" : "Port",
+        "description" : "Port of the database.",
+        "type" : "integer",
+        "minimum" : 0,
+        "maximum" : 65536,
+        "default" : 443,
+        "examples" : [
+          "443"
+        ],
+        "order" : 1
+      },
+      "database" : {
+        "title" : "DB Name",
+        "description" : "Name of the database.",
+        "type" : "string",
+        "order" : 2
+      },
+      "table" : {
+        "title" : "Default Table",
+        "description" : "The default  table was written to.",
+        "type" : "string",
+        "examples" : [
+          "default"
+        ],
+        "default" : "default",
+        "order" : 3
```

Comment:
The connector will sync only one table? Why not retrieve all tables in the database?

Reply:
Oh sorry, I just copy the spec.json from databend-destination, I will remove this.